packages/flutter_tools/test/general.shard/android/gradle_utils_test.dart

```diff
@@ -779,7 +779,7 @@ dependencies {
                 gradleV: data.gradleVersion,
                 agpV: data.agpVersion,
               ),
-              data.validPair ? isTrue : isFalse,
+              data.validPair,
```

Comment:
Why this change? isTrue I think gives a better error message than comparing to true.

Reply:
Ah I think in my skimming I read this as a case of `boolean ? true : false`, my mistake. I'll change back